R/translator.R

```diff
@@ -83,6 +87,8 @@ Translator$methods(
     },
     translate = function(keyword) {
       "Translates 'keyword' to language specified by 'set_translation_language'"
+      if (isTRUE(automatic))
+        stop("For automatic translations use 'automatictranslate' or 'at'.")
```

Comment:
Does it mean we cannot use both custom translations and google ones? I think it's an unnecessary restriction. WDYT?

Reply:
agreed